moped-editor/src/views/projects/projectView/TabPanel.js

```diff
@@ -15,7 +15,7 @@ function TabPanel(props) {
     >
       {value === index && (
         <Box p={3} className={props.className}>
-          <Typography>{children}</Typography>
+          {children}
```

Comment:
This change impacts every tab of the `ProjectView`, but as far as I can tell it only actually affects the `DataTable` component, which is only in use on `ProjectSummary`